pkg/config/config.go

```diff
@@ -7,36 +7,55 @@ import (
 	"strings"
 	"unicode"
 
-	"github.com/rs/zerolog/log"
+	"github.com/rs/zerolog"
 	flag "github.com/spf13/pflag"
 	"k8s.io/client-go/util/homedir"
 )
 
+type ZeroLogLevel int8
+
+func (l ZeroLogLevel) String() string {
+	return zerolog.Level(l).String()
+}
+
+func (l *ZeroLogLevel) Set(s string) error {
+	level, err := zerolog.ParseLevel(s)
+	if err != nil {
+		return err
+	}
+	*l = ZeroLogLevel(int8(level))
+	return nil
+}
+
+func (l ZeroLogLevel) Type() string {
+	return "string"
+}
+
 type Config struct {
 	AdditionalKinds []string
 	Cluster         bool
-	Debug           bool
 	ExitError       bool
 	Filenames       []string
 	Helm2           bool
 	Helm3           bool
 	Kubeconfig      string
+	LogLevel        ZeroLogLevel
 	Output          string
 }
 
 func NewFromFlags() (*Config, error) {
-	config := Config{}
+	config := Config{LogLevel: ZeroLogLevel(int8(1))}
```

Comment:
Same, just:
```suggestion
	config := Config{LogLevel: 1}
```
or imho more readable:
```suggestion
	config := Config{LogLevel: ZeroLogLevel(zerolog.InfoLevel)}
```